app/models/samples/schedule_ruleset.py

```diff
@@ -0,0 +1,290 @@
+schedule_ruleset = {
+    'type': 'ScheduleRuleset',
+    'name': 'Schedule Ruleset',
+    'schedule_type_limits': {
+        'type': 'ScheduleTypeLimits', 
+        'numeric_type': {
+            'numerictype': {
+                'type': 'ScheduleContinuous'
+            }  
+        },
+        'unit_type': 'Temperature',
+        'name': 'Numeric Type',
+        'lower_limit_value': 0,
+        'upper_limit_value': 20    
+    }, 
+    'default_day_schedule': [
+        {
+            'type': 'ScheduleDay',
+            'name': 'Default Day 1',
+            'interpolate_to_timestep': 'No',
+            'hour': 23,
+            'minute': 59,
+            'value_until_time': 20
+        }
+    ],
+    'summer_designday_schedule': [
+        {
+            'type': 'ScheduleDay',
+            'name': 'Default Day 1',
+            'interpolate_to_timestep': 'No',
+            'hour': 23,
+            'minute': 59,
+            'value_until_time': 20
+        }
+    ],
+    'winter_designday_schedule': [
+        {
+            'type': 'ScheduleDay',
+            'name': 'Default Day 1',
+            'interpolate_to_timestep': 'No',
+            'hour': 23,
+            'minute': 59,
+            'value_until_time': 20
+        }
+    ],
+    'schedule_rule': [
+        {
+            'type': 'ScheduleRule',
+            'schedule_day': [
+                {
+                    'type': 'ScheduleDay',
+                    'name': 'Default Day 1',
+                    'interpolate_to_timestep': 'No',
+                    'hour': 23,
```

Comment:
In OpenStudio and EnergyPlus the last value is until 24:00

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/openstudio_app/Resources/default/office_default.osm#L171